### ISSUE TITLE
[Feature Request] Show how many missions that are in the queue.

### ISSUE THREAD

**Issue by zeenk** [TLDRMissions/TLDRMissions#335]
**TLDR Version of your idea**
TLDR version: Title

**Slightly longer version**
Would be nice to have a small indicator that shows how many missions that are going to be attempted to get sent out, if you are at lets say mission 5/8 in the queue and you don't have that many followers left to work with and it's a pretty difficult mission to beat that require some extra followers, you could easier tell if you should skip it and try for the other remaining missions instead that might only need 1-2 followers + troops to beat so you get more value out of your followers :)

**Additional context**
Example:
![169675467-28993a9b-f017-4f5b-92e8-e8fccd78776a](https://user-images.githubusercontent.com/68170439/171696206-fdcf71d1-bea6-4534-aecb-3cdbef29713d.png)

**Comment by teelolws** [comment]
Coming next patch.
